### ISSUE TITLE
How to use apply_optics() to introduce the phase of the optical_element

### ISSUE THREAD

**Issue by Zhengjle1** [LASY-org/lasy#268]
In the 'API documentations' of the introduction of LASY,  I want to know how to use the function apply_optics() which I can't find examples of how to use it. Please let me know the basic usage of this function. Thanks!

**Comment by RemiLehe** [comment]
Agreed, we should add a tutorial on optical elements in the documentation.

For now, you can have a look at this example:
https://github.com/LASY-org/lasy/blob/development/tests/test_parabolic_mirror.py

(although it is not in the documentation)
